press-this.php

```diff
@@ -744,20 +744,20 @@ public function serve_app_html() {
 		<div class="options-panel">
 			<div class="post-options">
 				<?php if ( $supports_formats ) : ?>
-					<a href="#" class="post-option">
+					<a href="#" class="post-option" tabindex="0">
```

Comment:
Didn't realize! :smile: I'll remove it. BTW, we should do something with that `href="#"`, right? I suppose this is the case of a `<button>`?